packages/config/src/common.ts

```diff
@@ -24,13 +24,39 @@ export type Schema = {
 };
 
 export type Table = {
+  /**
+   * Human-readable label for this table. Used as config keys, library names, and filenames.
+   * Labels are not length constrained like resource names, but special characters should be avoided to be compatible with the filesystem, Solidity compiler, etc.
```

Comment:
Wonder if these special characters should be specified here? Curious if there are any other constraints like can a label start with number, etc?

Reply:
I don't know the range of special characters in various OS's so I'll leave it to user discretion

if a label has a non-fs compatible character, they'll know quickly because tablegen/worldgen will fail